projects/giflib/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "http://giflib.sourceforge.net/"
+primary_contact: "esr@thyrsus.com"
+auto_ccs:
+  - "vincent.ulitzsch@live.de"
+sanitizers:
```

Comment:
I think you can use all sanitizers, the target should be working fine, just remove lines 5-6.

Reply:
Unless you have reasons to use ASan only.